solutions/instances/main.tf

```diff
@@ -121,16 +178,51 @@ module "cos" {
       metrics_monitoring_crn  = var.existing_monitoring_crn
     }
   }]
+}
+
+module "cos" {
+  providers = {
+    ibm = ibm.cos
+  }
+  count                    = var.existing_scc_cos_bucket_name == null && var.existing_scc_instance_crn == null ? 1 : 0 # no need to call COS module if consumer is passing existing SCC instance or COS bucket
+  source                   = "terraform-ibm-modules/cos/ibm//modules/fscloud"
+  version                  = "8.11.14"
```

Comment:
@kierramarie the problem is you are reverting to an older COS module version. Main branch is on `8.14.1`. Please update